driver/userspace.c

```diff
@@ -650,7 +650,7 @@ WnbdParseUserIOCTL(PWNBD_EXTENSION DeviceExtension,
         if (!Irp->AssociatedIrp.SystemBuffer ||
             CHECK_O_LOCATION_SZ(IoLocation, RequiredBuffSize))
         {
-            WNBD_LOG_ERROR("IOCTL_WNBD_LIST: Bad output buffer");
+            WNBD_LOG_WARN("IOCTL_WNBD_LIST: Bad output buffer");
```

Comment:
Let's use a debug message here. Each list command will get a warning since we're adjusting the buffer size.